docker/webui/run_openqa.sh

```diff
@@ -9,5 +9,5 @@ dbus-daemon --system --fork
 start_daemon -u geekotest /usr/share/openqa/script/openqa-scheduler &
 start_daemon -u geekotest /usr/share/openqa/script/openqa-websockets &
 start_daemon -u geekotest /usr/share/openqa/script/openqa gru -m production run &
-rcapache2 start
+apache2ctl start
```

Comment:
I don't know - I was just thinking aloud. But now that you mention it (not sure why @wnereiz ignored my thought :), the rest of this bash script doesn't look like it's handled by systemd either.

So just merge and take the blame ,)